src/server/auth/server/testing/auth_test.go

```diff
@@ -2603,10 +2603,13 @@ func TestDebug(t *testing.T) {
 		require.NoError(t, err)
 		expectedFiles[pattern] = g
 	}
-	pattern := path.Join("input-repos", dataRepo, "commits")
-	g, err := globlib.Compile(pattern, '/')
-	require.NoError(t, err)
-	expectedFiles[pattern] = g
+	// Record glob patterns for expected source repo files.
+	for _, file := range []string{"commits", "commits-chart**"} {
```

Comment:
Why was this added?

Reply:
We now collect commit charts for source repos, so we are checking for those existing in the debug dump output. The reason for the `**` at the end is because the expectation is to produce an error file when collecting the chart in this workload because there is only one commit.